test/v2/client/test_client.go

```diff
@@ -661,23 +626,21 @@ func (c *TestClient) ReadBlobFromValidatorsInQuorum(
 		defer cancel()
 	}
 
+	start := time.Now()
+
 	if validateAndDecode {
-		start := time.Now()
 
 		retrievedBlobBytes, err := c.validatorClient.GetBlob(
 			ctx,
-			blobKey,
-			blobVersion,
-			blobCommitments,
-			uint64(referenceBlockNumber),
-			quorumID)
+			header,
+			uint64(referenceBlockNumber))
 		if err != nil {
 			return fmt.Errorf("failed to read blob from validators, %s", err)
 		}
 
-		c.metrics.reportValidatorReadTime(time.Since(start), quorumID)
+		c.metrics.reportValidatorReadTime(time.Since(start), 0)
```

Comment:
optionally, you could remove the `quorumID` parameter from `reportValidatorReadTime` entirely, since this is the only usage of that method, and `quorumID` is no longer relevant